src/bouwdossiers/migrations/0007_alter_bouwdossier_options.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.1.13 on 2024-02-13 13:24
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bouwdossiers', '0006_auto_20201228_1449'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='bouwdossier',
+            options={},
+        ),
+    ]
```

Comment:
Misschien deze verwijderen en makemigrations opnieuw runnen om te kijken wat er gebeurt

Reply:
Results in the same migration